code/modules/mob/new_player/login.dm

```diff
@@ -14,19 +14,16 @@
 		loc = locate(1,1,1)
 	lastarea = loc
 
+	client.screen = list() // Remove HUD items just in case.
+	client.images = list()
+	if(!hud_used)
```

Comment:
Ideally we'd call ..() here and have /mob/Login() handle this like /tg/ does, but I don't want to risk that breaking other stuff, as new_players are /mob/dead/new_player() on /tg/.

Reply:
Yeah, that's one change we need to make, some time.

Killing off unecessary mob types

`/mob/living/carbon/slime` -> `/mob/living/simple_animal/slime`
`/mob/living/carbon/brain` ->`/mob/living/brain`

amongst the more obvious ones like new players and the various dead types.

Not really a priority though.